packages/shared/src/components/plus/PlusComparingCards.tsx

```diff
@@ -57,6 +58,7 @@ const PlusCard = ({
 }: PlusCardProps): ReactElement => {
   const id = useId();
   const { logSubscriptionEvent } = usePlusSubscription();
+  const { isFreeTrialExperiment } = usePaymentContext();
```

Comment:
Should this be not coming from the prop? This would mean all PlusCard, if the user is under the experiment, would display the Free trial one. If this is not going to be used anywhere outside free trial for enrolled users, then feel free to ignore this.

Reply:
Only use is in onboarding, if user have been enrolled in experiment we need to show always the changed label. I can move this upside but basically won't change much. Let me know what do you prefer 👍